src/main.rs

```diff
@@ -63,11 +63,11 @@ static LOGGER: Logger = Logger;
 struct Logger;
 
 impl log::Log for Logger {
-  fn enabled(&self, metadata: &log::Metadata) -> bool {
+  fn enabled(&self, metadata: &log::Metadata<'_>) -> bool {
     metadata.level() <= log::max_level()
   }
 
-  fn log(&self, record: &log::Record) {
+  fn log(&self, record: &log::Record<'_>) {
```

Comment:
Good question, I'd thought they look a little weird. I'll try and remove them and see if everything compiles.

It could be a bug with `cargo fix`...

Reply:
Looks like these are not required, I will remove them.

Reply:
Actually I am not sure about this, so holding back my commit.

Perhaps someone else else can chime in. 
> The behavior around dyn Trait is probably worth highlighting, since it is one case where '_ differs from writing nothing at all.

From this issue: https://github.com/rust-lang/rust/issues/48469

Reply:
Okay, it looks like it is explained here: https://rust-lang-nursery.github.io/edition-guide/rust-2018/ownership-and-lifetimes/the-anonymous-lifetime.html